data_structures/linked_list/doubly_linked_list.py

```diff
@@ -205,23 +241,12 @@ def create_linked_list():
     >>> new_linked_list.get_head_data()
     20
 
-    >>> new_linked_list_2 = LinkedList()
-    >>> for value in range(1,10):
-    ...    new_linked_list_2.insert(value=value)
-    >>> print(new_linked_list_2)
```

Comment:
Let's not lose this test!  We want a test of a linked list that is longer than one element.